.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+
+env:
+  global:
+    - LUAJIT_LIB=/usr/lib64/libluajit-5.1.so
+    - LUAJIT_INC=/usr/include/luajit-2.0
+    - LUA_INCLUDE_DIR=/usr/include/luajit-2.0
```

Comment:
@chipitsine Well, we can just build it from source via the `v2.1-agentzh` branch of openresty's luajit2 repo:

https://github.com/openresty/luajit2/tree/v2.1-agentzh